integ-test/src/test/resources/indexDefinitions/text_keyword_index_mapping.json

```diff
@@ -0,0 +1,33 @@
+{
+  "mappings" : {
+    "properties" : {
+      "typeKeyword" : {
+        "type" : "keyword"
+      },
+      "typeText" : {
+        "type" : "text"
+      },
+      "typeKeywordFieldNoFieldData" : {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }      },
+      "typeTextFieldData" : {
+        "type": "text",
+        "fielddata": true,
```

Comment:
it would be interesting to know how it works if we have a `text` field with `"fielddata": true` but doesn't have any `fields` defined.  Does this behave just like `typeTextFieldData`?

Reply:
Added in 2a8fbacd599f31607935ba849ec4cf89527d9fb5.